joomla/joomla-cms
# [4.2] Fix meta description for article view

Pull Request for Issue #39587.

### Summary of Changes
This PR fixes wrong article meta-description priority as described in https://github.com/joomla/joomla-cms/issues/39587. Basically, it should work as below:

1. If the menu item is linked directly to the article, the meta description set in menu item will take highest priority
2. If not, the meta description from article will be used
3. And finally, the meta description from menu item

### Testing Instructions
1. Follow instructions at https://github.com/joomla/joomla-cms/issues/39587 , confirm the issue
2. Apply patch, confirm that the issue fixed


### Actual result BEFORE applying this Pull Request
Meta description is not handled properly.


### Expected result AFTER applying this Pull Request
Meta description is handled properly.


### Link to documentations
Please select:
